### PR TITLE
Add Dex Finance tokens similar to commit #1393

### PR DESCRIPTION
**Pull request by Sevik-The-Degen**
Brief comments on the purpose of your changes:

*For Dune Engine V2*
I've checked that:

* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

When you are ready for a review, tag duneanalytics/data-experience. We will re-open your forked pull request as an internal pull request. Then your spells will run in dbt and the logs will be avaiable in Github Actions DBT Slim CI. This job will only run the models and tests changed by your PR compared to the production project.

### CODE REVIEW

**Comment by Sevik-The-Degen** [comment]
@duneanalytics/data-experience

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -85,7 +85,9 @@ VALUES
     ("comp-compoundd", "bnb", "COMP", "0x52ce071bd9b1c4b00a0b92d298c512478cad67e8", 18),
     ("cummies-cumrocket", "bnb", "CUMMIES", "0x27ae27110350b98d564b9a3eed31baebc82d878d", 18),
     ("dai-dai", "bnb", "DAI", "0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3", 18),
-    ("dexe-dexe", "bnb", "DEXE", "0x039cb485212f996a9dbb85a9a75d898f94d38da6", 18),
+    ("dex-dexira", "bnb", "DEX", "0x147E07976E1ae78287c33aAFAab87760d32E50A5", 18),
+    ("dexe-dexe", "bnb", "DEXE", "0x039cb485212f996a9dbb85a9a75d898f94d38da6", 18),
```
Comment (jeff-dude) [inline models/prices/prices_tokens.sql:89]: this appears to be ethereum token: https://api.coinpaprika.com/v1/coins/dexe-dexe

Comment (jeff-dude) [reply models/prices/prices_tokens.sql:89]: also, i believe all new entries should have contract address lowercase. maybe @aalan3 can help confirm

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (aalan3) [reply models/prices/prices_tokens.sql:89]: This is correct @jeff-dude. In general all contract addresses (except for solana ones) in v2 need to be lowercase.

**Comment by jeff-dude** [comment]
@Sevik-The-Degen please see comments above on findings and how to move forward

**Comment by jeff-dude** [comment]
closing due to inactivity. please do reopen a PR with updated changes when ready 👍

**Pull request closed without merging**